modules/plugin/iau-wkt/src/test/java/org/geotools/referencing/iau/IAUExtensionTest.java

```diff
@@ -107,17 +105,27 @@ public void test1000() throws FactoryException {
         assertNotNull(identifiers);
         assertFalse(identifiers.isEmpty());
 
-        String asString = identifiers.toString();
-        assertTrue(asString, identifiers.contains(new NamedIdentifier(Citations.IAU, "1000")));
+        // citation is IAU but not Citations.IAU...
+        // String asString = identifiers.toString();
+        // assertTrue(asString, identifiers.contains(new NamedIdentifier(Citations.IAU, "1000")));
```

Comment:
I assume that these comments should stay here so once things work as expected they can get restored?

Reply:
I can actually uncomment that, it works now. Thanks for finding it.